ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -359,21 +372,31 @@ private void initializeInstances(List<InstanceConfig> instanceConfigs) throws Ex
     private void updateStateOfReplicas(List<InstanceConfig> instanceConfigs) {
```

Comment:
Can you update the java doc of both `updateStateOfReplicas` and `initializeInstances` method?

Reply:
Done.